app/renderer/api/shortcuts.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const Mousetrap = require('mousetrap');
+
+const AppStatus = require('../app_status');
+const GauchoActions = require('./gaucho_actions');
+const Material = require('./materialize');
+
+// Methods interface with jQuery
+module.exports = {
+    init() {
+        this.changeTabForward();
+        this.changeTabBack();
+        this.toggleEdit();
+        this.toggleNavBarMenu();
+    },
+    changeTabForward() {
+        Mousetrap.bind('ctrl+tab', () => {
+            const totalTabs = Material.getLengthNavBarTabs() - 1
+            const newIndex = AppStatus.activeSuite === totalTabs
```

Comment:
This code is a bit hard to read (using a ternary operator when the condition is not simple makes it hard to read, maybe an if is more readable?

If not, please, write a single line for the operator